common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -146,4 +153,18 @@
             </ul>
         }
     </div>
+
+    <div class="menu-brand-extensions hide-until-desktop">
```

Comment:
Can we hide the duplicate values in the menu on Desktop? We could potentially add a class for that.